apps/accounts/models.py

```diff
@@ -71,10 +72,44 @@ class SpotifyUserAuth(BaseModel):
     access_token = EncryptedCharField(max_length=100)
     refresh_token = EncryptedCharField(max_length=100)
     last_refreshed = models.DateTimeField(auto_now_add=True)
+    top_artists = ArrayField(models.CharField(max_length=200), default=list)
```

Comment:
Let's move this data to a separate model called `SpotifyUserData`. It makes more sense to store the data for a user from Spotify in a separate table, as it is different that the data we use to authenticate a user with Spotify. The table should have a one to one field mapping a `SpotifyUserAuth` to a `SpotifyUserData` record.